best-practices/README.md

```diff
@@ -13,6 +13,12 @@ General
 * [Keep the code simple](http://www.readability.com/~/ko2aqda2).
 * Avoid premature optimizations. Get a version 1.0 out there as soon as you
   can. Until you have some users to measure, you're optimizing based on guesses.
+* Use `bin/setup` for initial setup of the project and for development
```

Comment:
One rule of thumb I really liked about this is, "after running it the tests should be green". We could add that here.

Reply:
Like that detail. Added.